app/jobs/cache/record_counts/providers_job.rb

```diff
@@ -9,19 +9,31 @@ class ProvidersJob < ActiveJob::Base
 
       queue_as :default
 
-      def perform
-        params = { query: '*:*', rows: 0, profile: 'minimal facets', facet: 'PROVIDER' }
-        response = repository.search(params)
+      def perform(collection_id = nil)
```

Comment:
Assignment Branch Condition size for perform is too high. [25.79/15]<br>Method has too many lines. [21/10]